kie-osgi/kie-karaf-features/pom.xml

```diff
@@ -47,9 +47,11 @@
     <karaf.version.org.apache.servicemix.bundles.ant>1.8.2_2</karaf.version.org.apache.servicemix.bundles.ant>
     <karaf.version.org.apache.servicemix.bundles.dom4j>1.6.1_5</karaf.version.org.apache.servicemix.bundles.dom4j>
     <karaf.version.org.apache.servicemix.bundles.serp>1.14.1_1</karaf.version.org.apache.servicemix.bundles.serp>
-    <karaf.version.com.fasterxml.classmate>1.3.1</karaf.version.com.fasterxml.classmate>
+    <karaf.version.com.fasterxml.classmate>1.5.1</karaf.version.com.fasterxml.classmate>
     <karaf.version.org.jboss.jandex>2.2.3.Final</karaf.version.org.jboss.jandex>
     <karaf.version.javax.validation>1.1.0.Final</karaf.version.javax.validation>
+    <karaf.version.org.javassist>3.27.0-GA</karaf.version.org.javassist>
+    <karaf.version.org.hibernate.commons.annotations>5.1.2.Final</karaf.version.org.hibernate.commons.annotations>
```

Comment:
Apart from H2 version, I had to specify these versions to meet the hibernate-core version `5.4.24.Final`'s dependencies.